src/framework/mlt_tractor.c

```diff
@@ -345,6 +345,11 @@ mlt_producer mlt_tractor_get_track(mlt_tractor self, int index)
     return mlt_multitrack_track(mlt_tractor_multitrack(self), index);
 }
 
+const char *mlt_tractor_passthrough_properties()
```

Comment:
Since this is a public function, it will require documentation. I am not sure what the documentation should say.  In fact, since this function is such a nuanced detail, I think that we should not add it. We can just duplicate the properties string everywhere it needs to be used.